test/transactionBreakdown.test.ts

```diff
@@ -0,0 +1,57 @@
+import { connectDatabase, disconnectDatabase } from '../utils/test-utils/db.util';
+import supertest from 'supertest';
+import testApp from '../app';
+const request = supertest(testApp);
+
+import transactionBreakdownSeed from './transactionBreakdown.test.data';
+
+describe('transactionBreakdown routes', () => {
+  beforeAll(() => {
+    connectDatabase();
+  });
+
+  afterAll(() => {
+    disconnectDatabase();
+  });
+
+  describe('POST route', () => {
+    it('should return a transactionBreakdown with accurate orderTransaction', async () => {
+      const res = await request.post('/api/transactionBreakdown').send(transactionBreakdownSeed[0]);
```

Comment:
Would be clearer to assign each member of the transactionBreakdownSeed array to a variable naming that identifies what's the dataset for. So for `transactionBreakdownSeed[0]` maybe something like, validTransactionBreakdownInput. For `transactionBreakdownSeed[1]` transactionBreakdownInputWithZeroTip, etc.